16/umbraco-cms/customizing/extending-overview/extension-types/kind.md

```diff
@@ -33,7 +29,7 @@ To register a Kind, use the same method as other extensions. The key properties
 The following example shows how to register a Button Kind for [**Header Apps**](../extension-types/header-apps.md). This kind provides a preset configuration for a button element that can be reused by other Header App extensions.
 
 ```typescript
-const manifest: ManifestKind = {
+const manifest: ManifestKind<UmbExtensionManifest> = {
```

Comment:
I think you can use `UmbExtensionManifestKind` here instead of `ManifestKind<UmbExtensionManifest>`.  Not tested, but that's what I see when I look in the core code.